File: src/_6kyu/_6kyu_highest_scoring_word/HighestScoringWordTest.java
package _6kyu._6kyu_highest_scoring_word;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class HighestScoringWordTest {

    @Test
    void sampleTests() {
        assertEquals("taxi", HighestScoringWord.high("man i need a taxi up to ubud"));
        assertEquals("volcano", HighestScoringWord.high("what time are we climbing up to the volcano"));
        assertEquals("semynak", HighestScoringWord.high("take me to semynak"));
    }

}

File: src/_6kyu/_6kyu_highest_scoring_word/HighestScoringWordTest.cs
namespace Solution {
	using NUnit.Framework;
	using System;
	using System.Collections.Generic;

	[TestFixture]
	public class Sample_Tests {
		private static IEnumerable < TestCaseData > testCases {
			get {
				yield
				return new TestCaseData("man i need a taxi up to ubud").Returns("taxi");
				yield
				return new TestCaseData("what time are we climbing up to the volcano").Returns("volcano");
				yield
				return new TestCaseData("take me to semynak").Returns("semynak");
			}
		}

		[Test, TestCaseSource("testCases")]
		public string Test(string s) = >Kata.High(s);
	}
}